README.md

```diff
@@ -1,56 +1,108 @@
+
+```bash
+ml SQLite 
+sqlite3 /groups/${GROUP}/${TMP}/trendanalysis/database/chronqc_db/chronqc.stats.sqlite
+.table (see which tables are in the database)
+select * from ${table}; (see what kind of data is in the table)
+.exit (to exit sqlite)
+```
+
 When a project or other type of run is added to the database, a line is added to a log file containing the project or data type. 
-These log files are in `/groups/${GROUP}/${TMP}/logs/trendanalysis/`.
+These log files are in `/groups/umcg-gd/${TMP}/logs/trendanalysis/`.
+
+If you want to generate the database again, remove the database and the log files.
+Run trendAnalyse.sh again, all datatypes are added which are on ${TMP}.
+If you want to test with 1 datatype, see https://chronqc.readthedocs.io/en/latest/run_chronqc.html.
+You can run chronQC with the runDateInfo file and the table file.
+You can play with the different table types. Adjust the .json file you use to generate the plots. The *.json files are in the /template/ folder in the Trendanalysis repo.
 
 Then, in the trendAnalyse.sh script, the file templates/reports.sh is sourced.
-In this file the commands are noted so you can generate the reports files you like, for the data types you want.
-In the command to generate the reports a json file is needed. These are also in the /templated/ folder.
+In this file the commands are noted so you can generate the report files you like, for the data types you want.
+In the command to generate the reports a json file is needed. These are also in the /templates/ folder in the Trendanalysis repo.
 In these json files you can note which type of graphs you want to see in the report, which is in html file format.
 
+Side note: for instance the inhouse rawdata sequence runs: The data of all sequencers are in 1 table in the database, but a selection is made based on the number of the sequencer. 
+Each sequencer has it's own end report. 
+
 ## step 3: copyTrendAnalysisDataToPrm.sh
 
 The `copyTrendAnalysisDataToPrm.sh` copies all the reports back to PRM, where the diagnostics can take a look at the QC data over time.
 
+TMP location: `/groups/${GROUP}/${TMP}/trendanalysis/reports/${date}/chronqc_output/`
+
+PRM location: `/groups/${GROUP}/${PRM}/trendanalysis/reports/${date}/chronqc_output/`
 
-## important
-When the database is created new again. It is important to know for the inhouse NGS projects to first add a new project, generated with the NGS_DNA 4.2.2, or newer.
-And then add all the older projects. The multiQC version in the NGS_DNS-4.2.2 is updated. The QC data generated has different columns, compared to an older version.
-If old date was first added to the database, the newer data can't be added because columns are missing. ChronQC can't handle this.
+## Important
+When creating a new database for production, it is important to know that a new inhouse NGS project (generated with NGS_DNA-4.2.2 or newer) is added first, 
+this is due to different columns in comparison with an older version. 
```

Comment:
```suggestion
When creating a new database, a new inhouse NGS project (generated with NGS_DNA-4.2.2 or newer) must be added first, because the newer data contains different columns in comparison to data from older versions. 
```